src/ui/controls/RadioListControl.tsx

```diff
@@ -40,6 +40,15 @@ export class RadioListControl extends React.Component<IStringControlProps, void>
   }
 
   /** @override */
```

Comment:
I've added these only for future-proofing compatibility with closure compiler per recommendation here: http://usejsdoc.org/tags-override.html